tests/foreman/api/test_errata.py

```diff
@@ -199,25 +179,23 @@ def test_positive_bulk_install_package(module_org, activation_key, custom_repo,
 
     :CaseLevel: System
     """
-    with VirtualMachine(distro=DISTRO_RHEL7) as client:
-        client.install_katello_ca()
-        client.register_contenthost(module_org.label, activation_key.name)
-        assert client.subscribed
-        client.enable_repo(REPOS['rhst7']['id'])
-        client.install_katello_agent()
-        host_id = entities.Host().search(query={'search': f'name={client.hostname}'})[0].id
-        _install_package(
-            module_org,
-            clients=[client],
-            host_ids=[host_id],
-            package_name=FAKE_1_CUSTOM_PACKAGE_NAME,
-            via_ssh=False,
-            rpm_package_name=FAKE_2_CUSTOM_PACKAGE,
-        )
+    rhel7_contenthost.install_katello_ca()
+    rhel7_contenthost.register_contenthost(module_org.label, activation_key.name)
+    assert rhel7_contenthost.subscribed
+    rhel7_contenthost.enable_repo(constants.REPOS['rhst7']['id'])
+    rhel7_contenthost.install_katello_agent()
```

Comment:
Can you implement a module-level fixture just to optimize these repeated steps? rather than using plain `rhel7_contenthost`, also for rhel6/8 clients if necessary.

Reply:
I'd prefer the tests to be more explicit in their behavior since it is more clear exactly what's going on.